### PR TITLE
Create javalinter.yml (#55)

### PR DESCRIPTION
**Pull request by lzoebaker**
* Create javalinter.yml

* Fixing java synatx with addition of new linter

### CODE REVIEW

File: src/main/java/com/google/sps/data/CareerQuestionDatabase.java [hunk 1]
```diff
@@ -22,37 +17,41 @@
...
-      Entity questionEntity = getQuestionDatastoreEntity(careerQuestionAndChoices);
-      this.datastore.put(questionEntity);
-      for (CareerQuestionChoice choice : careerQuestionAndChoices.getChoices()) {
-        Entity choiceEntity = getChoiceDatastoreEntity(question, choice);
-        this.datastore.put(choiceEntity);
-      }
-  } 
+  public void putCareerQuestionAndChoicesIntoDatabase(
+      CareerQuestionAndChoices careerQuestionAndChoices) {
+    String question = careerQuestionAndChoices.getQuestion();
```
Comment (lattai) [inline src/main/java/com/google/sps/data/CareerQuestionDatabase.java:41]: wonky indents, probably a whitespace goof

Comment (lzoebaker) [reply src/main/java/com/google/sps/data/CareerQuestionDatabase.java:41]: I think its a 4 space line continuation indent followed by a normal 2 space indent, linter *should* catch whitespace problems.

File: src/main/java/com/google/sps/data/CareerQuestionDatabase.java [hunk 2]
```diff
@@ -22,37 +17,41 @@
...
-      String question = careerQuestionAndChoices.getQuestion();
-      Entity questionEntity = getQuestionDatastoreEntity(careerQuestionAndChoices);
-      this.datastore.put(questionEntity);
-      for (CareerQuestionChoice choice : careerQuestionAndChoices.getChoices()) {
-        Entity choiceEntity = getChoiceDatastoreEntity(question, choice);
-        this.datastore.put(choiceEntity);
-      }
-  } 
+  public void putCareerQuestionAndChoicesIntoDatabase(
+      CareerQuestionAndChoices careerQuestionAndChoices) {
```
Comment (menynexi) [inline src/main/java/com/google/sps/data/CareerQuestionDatabase.java:40]: weird spaceing

Comment (lzoebaker) [reply src/main/java/com/google/sps/data/CareerQuestionDatabase.java:40]: 4 space line continuation

**Pull request merged**